svelte-kit-scss/src/routes/(authenticated)/[username]/[repo]/+layout.svelte

```diff
@@ -6,9 +6,9 @@
   const { repoInfo } = data;
 </script>
 
-<div class="container">
+<div>
   <RepoHeader repo={repoInfo} />
-  <div class="main">
+  <div class="main container">
```

Comment:
If you look at the `<style>` tag, you can see you are maybe having unintended changes.

Now there is no `.main` that is a descendent of `.container` and there is nothing styling or using the `.main` class anymore.

I think this looks fine though so you can probably just remove the `container` class entirely and the whole `<style>` tag, and there is an extra unneeded `<div>` in the template...  it can probably just be

```svelte
<script lang="ts">
  import RepoHeader from '$lib/components/Repo/RepoHeader/RepoHeader.svelte';
  import type { LayoutServerData } from './$types';

  export let data: LayoutServerData;
  const { repoInfo } = data;
</script>

<RepoHeader repo={repoInfo} />
<div class="container">
  <slot />
</div>

```

...and now that I am looking we don't use `repo` or `username` from the server data 🤷‍♂️